From 2b3a59ee72f87cc519b00de06929c9747e0f2e47 Mon Sep 17 00:00:00 2001
From: Bian Jin chen <kenjc.bian@rock-chips.com>
Date: Tue, 19 Nov 2019 19:14:46 +0800
Subject: [PATCH] net: rockchip_wlan: rtl8822bs: fix clang compile error

drivers/net/wireless/rockchip_wlan/rtl8822bs/core/rtw_debug.c:45:44: error: expansion of date or time macro is not reproducible [-Werror,-Wdate-time]
        RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);

Change-Id: I9f786a4305e668dac97d527baac4e948b53332ed
Signed-off-by: Bian Jin chen <kenjc.bian@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rtl8822bs/core/rtw_debug.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8822bs/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8822bs/core/rtw_debug.c
index 8874ef5120bf..80f52e640a8e 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8822bs/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8822bs/core/rtw_debug.c
@@ -42,7 +42,6 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
-- 
2.35.3

